src/trait_/common_ord.rs

```diff
@@ -87,16 +94,16 @@ pub fn build_ord_signature(item: &Item, trait_: &DeriveTrait, body: &TokenStream
 					})*
 				};
 
+				let path = trait_.path();
+				let method = match trait_ {
+					DeriveTrait::PartialOrd => quote! { partial_cmp },
+					DeriveTrait::Ord => quote! { cmp },
+					_ => unreachable!("unsupported trait in `prepare_ord`"),
+				};
+
 				// Nightly or unsafe (default) implementation.
 				#[cfg(any(feature = "nightly", not(feature = "safe")))]
```

Comment:
does this nesting still make sense?

Reply:
Removed while merging the other blocks like you said below.